### PR TITLE
fix: update audio panel for mobile landscape view

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/ChatMessageEditor.razor [hunk 1]
```diff
@@ -60,14 +61,13 @@
...
                 }
+                <div class="audio-panel-wrapper">
+                    <ChatAudioPanel Chat="@Chat" Class="top-chat-audio-panel"/>
+                </div>
+
             </div>
-            @if (ScreenSize.IsWide()) {
-                <ChatAudioPanel Chat="@Chat"/>
-            }
+            <ChatAudioPanel Chat="@Chat" Class="right-chat-audio-panel"/>
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/ChatMessageEditor.razor]: Why ChatAudioPanel is declared twice?

File: src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/ChatMessageEditor.razor [hunk 2]
```diff
@@ -28,6 +28,7 @@
         @if (ScreenSize.IsWide()) {
             <RelatedChatEntryPanel/>
         }
+        <div class="landscape-sub-footer"></div>
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/ChatMessageEditor.razor]: What 'landscape-sub-foote' is used for?

**Review by crui3er — CHANGES_REQUESTED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.css [hunk 3]
```diff
@@ -37,6 +45,10 @@
     @apply overflow-hidden;
     @apply break-all;
 }
+body.narrow .post-panel {
+    @apply landscape:py-0;
+    @apply landscape:bg-[var(--post-panel-narrow-landscape)];
```
Comment (frolyo) [inline src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.css]: do we really need to use arbitrary value approach here?

**Review by crui3er — APPROVED** [review]

File: src/nodejs/styles/colors.css [hunk 4]
```diff
@@ -762,6 +774,12 @@
     --recorder-on-middle: var(--magenta-50);
     --audio-panel-button: var(--white);
     --audio-panel-button-text: var(--gray-40);
+    --mobile-landscape-post-panel: var(--gray-30);
+    --mobile-landscape-audio-panel-button: var(--gray-30);
```
Comment (crui3er) [inline src/nodejs/styles/colors.css:778]: Where is this color used?

File: src/nodejs/styles/colors.css [hunk 5]
```diff
@@ -762,6 +774,12 @@
...
+    --mobile-landscape-audio-panel-button-text: var(--gray-60);
+    --mobile-landscape-audio-panel-playback-on: var(--blue-10);
```
Comment (crui3er) [inline src/nodejs/styles/colors.css:780]: Where is this colot used?

**Review by crui3er — APPROVED** [review]

File: src/nodejs/styles/colors.css (hunk 4 above)
Comment (iqmulator) [reply src/nodejs/styles/colors.css:778]: image
image

File: src/nodejs/styles/colors.css (hunk 5 above)
Comment (iqmulator) [reply src/nodejs/styles/colors.css:780]: image
image

**Pull request merged**